nss-day-cohort-44/rare-rest-api-rare-frosty-flakes
# Jb registration

# Description
Create a seed.sh file in your project directory
Place the code below in the file.
Run chmod +x seed.sh in the terminal.
Then run ./seed.sh in the terminal to run the commands.
#!/bin/bash
rm -rf levelupapi/migrations
rm db.sqlite3
python manage.py migrate
python manage.py makemigrations levelupapi
python manage.py migrate levelupapi
python manage.py loaddata users
python manage.py loaddata tokens
python manage.py loaddata gamers
python manage.py loaddata gametypes

## Issue ticket:

## Type of change

# Testing Instructions
`git fetch all`
`git checkout  jb-registration`
`python3 manage.py runserver`

Check to make sure you can register a new user and login

`git branch -d  jb-registration`

# Checklist:
- [ ] I have commented my code with at least "AUTHOR: FIRST LAST NAME" at the top, and then in hard-to-understand areas
